### PR TITLE
Fix image type field behaviour in image editing form

### PR DESCRIPTION
**Pull request by asaunier**
- Moderators can change to whatever type
- Non moderators can only change type from 'personal' to 'collaborative'. If type is 'collaborative' or 'copyright' the field is disabled.

Related to https://github.com/c2corg/v6_ui/issues/1441

### CODE REVIEW

File: c2corg_ui/static/js/editing/imageediting.js [hunk 1]
```diff
@@ -51,6 +51,11 @@ app.ImageEditingController = function($scope, $element, $attrs, $http,
    */
   this.exposureError = false;
 
+  /**
+   * @type {?string}
+   * @export
+   */
+  this.initial_image_type = null;
```
Comment (tsauerwein) [inline c2corg_ui/static/js/editing/imageediting.js]: `this.initialImageType`?

Comment (asaunier) [inline c2corg_ui/static/js/editing/imageediting.js]: I wanted to be consistent with the attribute in the model which is ``image_type``.

Comment (tsauerwein) [inline c2corg_ui/static/js/editing/imageediting.js]: Elsewhere `imageTypes` is used.

Comment (asaunier) [inline c2corg_ui/static/js/editing/imageediting.js]: OK :)
I will change to ``this.initialImageType`` if you recommend to do so.

**Pull request merged**